recipe/meta.yaml

```diff
@@ -6,19 +6,20 @@ package:
 
 source:
   url: https://github.com/SciTools/nc-time-axis/archive/v{{ version }}.tar.gz
-  sha256: 49c4b1022e601e22a2fb5fc6fc2af69070cceb5cdd01a880ae7379caef8f2f90
+  sha256: 2fae9b904a5284581aa8ac1fd954776fc246339665411724b23d0e3b4449332f
```

Comment:
I find this change a bit puzzling; why would the sha256 have changed?... but I suppose it is building so it should be fine.

Reply:
Really no idea... it confused me for ages

Reply:
I agree not a problem.

Although from some quick googling, it seems this is something to do with how gtihub generates the archive (see https://github.com/spack/spack/issues/5411)

Reply:
I recently had a similar problem (https://github.com/conda-forge/black-feedstock/pull/24#discussion_r643199600), which turned out to be because I generated the SHA from the GitHub file rather than the PyPI file. Don't know if that could be relevant here?

Reply:
> Although from some quick googling, it seems this is something to do with how gtihub generates the archive (see [spack/spack#5411](https://github.com/spack/spack/issues/5411))

Yeah, nice to know. My only though was that GH had changed something on their end... goodness knows enough time has passed for something like that to happen :+1:

Reply:
> I recently had a similar problem ([conda-forge/black-feedstock#24 (comment)](https://github.com/conda-forge/black-feedstock/pull/24#discussion_r643199600)), which turned out to be because I generated the SHA from the GitHub file rather than the PyPI file. Don't know if that could be relevant here?

We're okay to maintain pulling from GH and not PyPI... although `nc-time-axis` 1.3.0 will be using `setuptools-scm` and so we'll defo need to pull from PyPI instead, otherwise the recipe won't build :+1: